### PR TITLE
add blink_options to render, fix Blink testing

### PR DESCRIPTION
**Pull request by hhaensel**

### CODE REVIEW

**Comment by hhaensel** [comment]
Hm , no idea. I copied from of the Blink runtests. Maybe you have an idea.

**Comment by jmboehm** [comment]
Perhaps the fact that 

```
w = Window(Blink.Dict(:show => false), async=false);
``` 
appears outside of a `@testset` in [here](https://github.com/JuliaGizmos/Blink.jl/blob/master/test/runtests.jl) makes the difference. They say 

```julia
# IMPORTANT: Window(...) cannot appear inside of a @testset for as-of-yet
# unknown reasons.
```

**Comment by hhaensel** [comment]
I saw this, but we haven't put it in a testset, have we?

**Comment by hhaensel** [comment]
Maybe it's linked to the port numbers? For Genie testing I typically use a random port between 8081 and 9000.

File: test/runtests.jl [hunk 1]
```diff
@@ -17,7 +17,6 @@ tests = [
 
 for test in tests
     @testset "$test" begin
```
Comment (jmboehm) [inline test/runtests.jl:19]: This one means it's in a @testset, no?

**Comment by hhaensel** [comment]
True, you  got it! I had overlooked that loop!

**Pull request closed without merging**